### PR TITLE
Implement better function handling

### PR DESCRIPTION
**Pull request by bl-young**
Resolves issue #45

### CODE REVIEW

**Comment by bl-young** [comment]
@cclin130 could you confirm this works for the CEDA data?

File: import_emission_factors/generate_import_factors.py [hunk 1]
```diff
@@ -413,9 +412,20 @@ def process_mrio_data(year):
     '''
     Wrapper function to call correct MRIO processing function
     '''
-    if source == "exiobase":
-        process_exiobase(year_start=year, year_end=year, download=True)
-    # elif source == "foo":
+    source_fxn = config.get('process_function').split('/')
+    try:
+        module = __import__(source_fxn[0])
```
Comment (cclin130) [inline import_emission_factors/generate_import_factors.py]: this is very cool, TIL `__import___`

**Comment by cclin130** [comment]
@bl-young I'll pull the branch and give it a try

**Comment by cclin130** [comment]
@bl-young I pulled this branch and re-generated the import shares and import factors. On top of confirming everything runs, I checked the 2022 CEDA values and they are identical to what I had previously 🎉

**Pull request merged**